README.md

```diff
@@ -55,6 +58,18 @@ heif_file = pyheif.read("IMG_7424.HEIC")
 heif_file = pyheif.read(open("IMG_7424.HEIC", "rb").read())
 ```
 
+### Windows DLL Loading
+
+On Python 3.8+, heif.dll and its dependencies must be copied to the site-packages folder, or explicity located in script:
+
+```python
+import os
+os.add_dll_directory(r'X:\path\to\libheif\build')
+import libheif
```

Comment:
How is this possible to import `libheif` directly? Is it `pyheif`?

Reply:
Ooof, yes it should be pyheif; that was a lousy typo I made while genericizing it. Sorry.